dadjokes-onchain/foundry/.github/workflows/test.yml

```diff
@@ -0,0 +1,34 @@
+name: test
+
+on: workflow_dispatch
+
+env:
+  FOUNDRY_PROFILE: ci
+
+jobs:
+  check:
+    strategy:
+      fail-fast: true
+
+    name: Foundry project
```

Comment:
should the job name be "build and test foundry project"?

Reply:
This is the default from foundry, leaving as is